integrations/gitlab/gitlab_integration/gitlab_service.py

```diff
@@ -210,25 +218,29 @@ def get_filtered_groups_for_webhooks(
 
         return groups_for_webhooks
 
-    def create_webhooks(self, groups_for_webhooks) -> list[int | str]:
-        # Filter out groups that are not in the group mapping and creating webhooks for the rest
-        logger.info(
-            f"Creating webhooks for the groups: {[group.attributes['full_path'] for group in groups_for_webhooks]}"
-        )
+    def create_webhooks(self, groups_for_webhooks: list[tuple[Group, list[str]]]) -> list[int | str]:
         webhook_ids = []
-        for partial_group in groups_for_webhooks:
-            group_id = partial_group.get_id()
+        for group_to_events in groups_for_webhooks:
+            group = group_to_events[0]
+            events = group_to_events[1]
+            
+            # Filter out groups that are not in the group mapping and creating webhooks for the rest
+            logger.info(
+                 f"Creating webhooks for the groups: {group.attributes['full_path']}"
+            )
+            
+            group_id = group.get_id()
             if group_id is None:
                 logger.info(
-                    f"Group {partial_group.attributes['full_path']} has no id. skipping..."
+                    f"Group {group.attributes['full_path']} has no id. skipping..."
                 )
             else:
-                if self._is_exists(partial_group):
+                if self._is_exists(group):
```

Comment:
this method name needs to change, its internal code is referring to webhooks while the class name is GitlabService ( which is generic )

Reply:
suggestion, `_is_webhook_exists_for_group`

Reply:
cool, on it

Reply:
done